.circleci/config.yml

```diff
@@ -711,20 +715,53 @@ jobs:
     executor: node
     steps:
       - checkout
+      - build/determinator:
+          paths: "nms"
       - yarn-install
       - run:
           name: yarn test
           <<: *appdir
           command: yarn test:ci
       - magma_slack_notify
 
+  nms-e2e-test:
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      - NMS_ROOT=${MAGMA_ROOT}/nms/app/packages/magmalte
+    steps:
+      - checkout
+      - build/determinator:
+          paths: "nms"
+      - docker/install-dc
+      - run: sudo apt-get install -y apt-transport-https
+      - run: curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+      - run: echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+      - run: sudo apt-get update -y
+      - run: sudo apt-get install -y yarn
+      - run: echo 'export MAGMA_ROOT=$(pwd)' >> $BASH_ENV
+      - run:
+          command: |
+            source $NVM_DIR/nvm.sh
+            cd ${MAGMA_ROOT}/nms/app
+            nvm install stable
+            yarn install
+            cd ${MAGMA_ROOT}/nms/app/packages/magmalte
+            ./e2e_test_setup.sh
+      - store_artifacts:
+          path: /tmp/nms_artifacts
```

Comment:
What's that artifacts you're storing ?

Reply:
I store magmalte, mock server logs and screenshots of nms e2e test. 
https://app.circleci.com/pipelines/github/magma/magma/3293/workflows/a00e9bb4-ccc3-415c-886a-7a20799613e9/jobs/33441/artifacts